### PR TITLE
feat: added color theme

### PR DESCRIPTION
**Pull request by riyapojage**

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -246,4 +246,11 @@ a {
 	text-rendering: optimizeLegibility;
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+
+	--baground-primary: #F9F5F2,
```
Comment (starkjay21) [inline src/styles/index.css:250]: typo in background

File: tailwind.config.js [hunk 2]
```diff
@@ -12,45 +12,25 @@ export default {
...
 					foreground: 'hsl(var(--popover-foreground))',
 				},
-				card: {
-					DEFAULT: 'hsl(var(--card))',
-					foreground: 'hsl(var(--card-foreground))',
-				},
-			},
+				
+				colors: {
+					'color-primary': 'var(--baground-primary)',
```
Comment (starkjay21) [inline tailwind.config.js:22]: same typo

**Review by starkjay21 — COMMENTED** [review]
Please fix the typos

**Comment by Shurtu-gal** [comment]
Already done in #6

**Pull request closed without merging**